README.md

```diff
@@ -1,2 +1,60 @@
-# op-component-library
+# Welcome to OP-Components library 👋
```

Comment:
Rename to Welcome to One Platform Components Library

Reply:
update the README to include what this library is about and what design pattern it follows.

Reply:
@gisumit any ideas here?